java/ql/lib/semmle/code/java/frameworks/JavaIo.qll

```diff
@@ -13,6 +13,10 @@ private class JavaIoSummaryCsv extends SummaryModelCsv {
         "java.io;Writer;true;write;;;Argument[0];Argument[-1];taint",
         "java.io;Writer;true;toString;;;Argument[-1];ReturnValue;taint",
         "java.io;CharArrayWriter;true;toCharArray;;;Argument[-1];ReturnValue;taint",
+        "java.io;ObjectInput;true;read;;;Argument[-1];Argument[0];taint",
+        "java.io;DataInput;true;readFully;;;Argument[-1];Argument[0];taint",
+        "java.io;DataInput;true;readLine;();;Argument[-1];Argument[0];taint",
+        "java.io;DataInput;true;readUTF;();;Argument[-1];Argument[0];taint",
```

Comment:
Are those really correct? `readLine()` and `readUTF()` don't have any argument, shouldn't this be:
```suggestion
        "java.io;DataInput;true;readLine;();;Argument[-1];ReturnValue;taint",
        "java.io;DataInput;true;readUTF;();;Argument[-1];ReturnValue;taint",
```

Reply:
Absolutely. Thanks!